### ISSUE TITLE
Deprecate?

### ISSUE THREAD

**Issue by daurnimator** [mkottman/luacrypto#39]
It seems this is no longer maintained.
To prevent confusion, could you officially deprecate the library?

FWIW, I've now switched over to [luaossl](http://25thandclement.com/~william/projects/luaossl.html).

**Comment by mathieujobin** [comment]
I found this https://github.com/wahern/luaossl but it does not seems to provide the crypto dependency for lapis

**Comment by daurnimator** [comment]
Correct, it is a different library. However it has the same functionality/abilities

**Comment by greatwolf** [comment]
I wouldn't depreciate this until luaossl can actually build out-of-the-box on windows. It's not a suitable replacement otherwise.

**Comment by daurnimator** [comment]
@greatwolf have a look over at https://github.com/wahern/luaossl/issues/77

**Comment by mkottman** [comment]
If there is a luacrypto fork that's maintaned, or someone wants me to transfer ownership, let me know. As you may have noticed, I no longer have the time/ability to contribute. Otherwise I will just explicitly mark this project deprecated.

**Comment by greatwolf** [comment]
@mkottman thanks for getting back. If there are no other volunteers I wouldn't mind taking up the mantle. I have a lua project that does crypto trading/lending on various exchanges so I depend a lot on the hmac* functions for authentication so I have an interest in keeping luacrypto current.

How does the ownership transfer work? Do I need to remove my existing fork first?(it's actually a fork of a fork of this project)

**Comment by greatwolf** [comment]
@mkottman I just tested a dummy project out to see how ownership transfer works. It looks like I need to delete the fork under my account first before github will allow the transfer. I've already done that so you can do the ownership transfer when you get some time.

**Comment by daurnimator** [comment]
@greatwolf are you willing to maintain luacrypto though? There is quite a bit of code that needs updating.
I'd think it would be easier to migrate over to luaossl?

**Comment by greatwolf** [comment]
Yes, I don't mind putting some time into maintaining this. I get wanting to migrate over to luaossl but it just feels like it's making too many *six assumptions in the code. Keeping another ssl binding alternative in the ecosystem would be nice in case users have trouble getting one of them to work.

Actually, I already have the code changes on my local luacrypto repo to make it buildable on openssl 1.1.0. It would be nice to be able to commit it to upstream which is part of why I'm pushing to get this project maintained again.

It doesn't have to be me though, if there are other interested parties, luacrypto can be transfer into a organization group account with multiple collaborators. I'm not aware of any other takers atm though.

**Comment by depay** [comment]
@greatwolf I also need the compatible version with openssl 1.1.0 and made buildable changes locally. Could you supply a PR ?

**Comment by daurnimator** [comment]
luaossl now works on windows.

As there is still no one maintaining luacrypto, could we add the deprecation notice?
